samples/tiled-convolution/tiled-convolution.cpp

```diff
@@ -32,12 +32,7 @@
 #include <ctime>
 #include <iostream>
 
-#ifndef LOCAL_SIZE_M
-#define LOCAL_SIZE_M 16
-#endif
-#ifndef LOCAL_SIZE_N
-#define LOCAL_SIZE_N 16
-#endif
+// The LOCAL_SIZE_N and LOCAL_SIZE_M is set at compile time
```

Comment:
is set -> are set

Reply:
Why remove the ifndef checks? If you try to compile "manually", this will now stop working